aten/src/ATen/Declarations.cwrap

```diff
@@ -364,6 +364,7 @@
 [[
   name: _th_equal
   cpu_bool: True
+  cuda_bool: True
```

Comment:
this is orthogonal right?  I.e. can be in a separate PR, there's nothing that has to change as a result of comparison op return types changing.

Reply:
https://github.com/pytorch/pytorch/pull/21538